File: pokedex.c
//Pokedex in C
#include <stdio.h>
#include <string.h>


int main(void){
    char name[40];
    printf("Hello welcome to the Pokedex\n\n");
    printf("Please enter the Pokemon name you're looking for\n");
    scanf("%s", name);

    if(strcmp(name, "bulbasaur")==0){
        printf("Name: Bulbasaur \n");
        printf("Number: #0001 \n");
        printf("There is a plant seed on its back right from the day this Pokémon is born. The seed slowly grows larger.\n");


    }
}

File: Fun_projects/pokedex.py
#Pokedex python program

print("Hello welcome to the Pokedex")

name = input("Enter Pokemon Name:")
print("Pokemon is: " + name)
pokemons=[]

with open("pokedex.txt", 'r') as file:
    for line in file:
        pokemons.append(line.split(","))   
    
pokedex = {}
for line in pokemons:
    pokedex[line[1]] = line

try:
    print(pokedex[name])
except:
    print("Pokemon not available")
